draft-ietf-uuidrev-rfc4122bis.md

```diff
@@ -1239,7 +1239,7 @@ Fixed-Length Dedicated Counter Bits (Method 1):
 Monotonic Random (Method 2):
 : With this method, the random data is extended to also function as a counter.
   This monotonic value can be thought of as a "randomly seeded counter" which
-  MUST be incremented in the least significant position for each UUID created
+  SHOULD be incremented in the least significant position for each UUID created
```

Comment:
This entire thing is a suggestion, so I'm not sure it makes sense to use MUST here, it's sort of like saying "this is just advice and only way to solve the problem, but if you're listing, you MUST do it this way..." - I think a SHOULD makes more sense.

Reply:
The text is saying here that there are two ways to solve the problem. If the second choice is used and the counter isn't incremented, the problem isn't necessarily solved. I prefer the MUST.

Reply:
Okay cool, I'm all good with striking this change then and leaving it how it was (MUST).